integrations-and-supported-tools/azure-ml/README.md

```diff
@@ -0,0 +1,36 @@
+# AzureML how-to
```

Comment:
We should here use _Azure Machine Learning_, as "AzureML" is not an official term.